examples/tv-casting-app/tv-casting-common/src/CastingServer.cpp

```diff
@@ -25,6 +25,21 @@ using namespace chip::app::Clusters::ContentLauncher::Commands;
 
 CastingServer * CastingServer::castingServer_ = nullptr;
 
+CastingServer::CastingServer()
+{
+#if CHIP_ENABLE_ROTATING_DEVICE_ID && defined(CHIP_DEVICE_CONFIG_ROTATING_DEVICE_ID_UNIQUE_ID)
+    // generate and set a random uniqueId for generating rotatingId
+    uint8_t rotatingDeviceIdUniqueId[chip::DeviceLayer::ConfigurationManager::kRotatingDeviceIDUniqueIDLength];
+    for (size_t i = 0; i < sizeof(rotatingDeviceIdUniqueId); i++)
+    {
+        rotatingDeviceIdUniqueId[i] = chip::Crypto::GetRandU8();
+    }
+
+    MutableByteSpan rotatingDeviceIdUniqueIdSpan(rotatingDeviceIdUniqueId);
```

Comment:
```suggestion
    ByteSpan rotatingDeviceIdUniqueIdSpan(rotatingDeviceIdUniqueId);
```
Because callee has no reason to modify it.

Reply:
fixed